### DISCUSSION TITLE
Fusermount of repository not working in Unraid

### DISCUSSION THREAD

**Discussion by R4z3Rx** [garethgeorge/backrest#450] [Q&A]
Hello, 'm trying to use backrest (in a docker container on my Unraid server) as the main backup software for my server to an offsite location. 
The backup and restore seem to be working (tried a partial restore) from the s3 repository (MinIO).
Yet if I try to mount the repository (would like to browse and easily recover selected files), I encounter the following error:

```
command: /bin/restic-0.17.0 mount repos -o sftp.args=-oBatchMode=yes
fusermount: exec: "fusermount": executable file not found in $PATH
[unknown] exit status 1
```

Would you be able to advise on how to fix it?

**Comment by garethgeorge** [comment]
Hey, fusermount unfortunately isn't supported in Backrest's docker container. You can use the mount command from backrest installed on baremetal (e.g. using the install script) but it comes with a number of footguns. Most significant is that your command is killed when you navigate away from Backrest.

**Reply by R4z3Rx** [reply]
Thanks for the reply and loving your work! Any possibility/ plan it will be supported/ integrated in the future?
Thanks
